controllers/topics/index.js

```diff
@@ -441,8 +487,8 @@ const getFollowerList = async (req, res) => {
 module.exports = {
   topics,
   checkName,
-  create,
   inviteMembers,
+  createTopic,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The export of `createTopic` is added here. Ensure that all places where this module is imported are updated to use `createTopic` instead of the old `create`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->